packages/cactus-cmd-socketio-server/package.json

```diff
@@ -26,7 +26,8 @@
     "socket.io": "^2.0.4",
     "ts-node": "8.9.1",
     "web3": "^1.2.9",
-    "xmlhttprequest": "^1.8.0"
+    "xmlhttprequest": "^1.8.0",
```

Comment:
@brioux Are these dependencies needed in this package? I didn't see changes in the source code here. (only did a rudimentary first pass so just asking)

Reply:
@petermetz no they are not

Reply:
@brioux Cool, please remove all the unused dependencies that were added to any of the packages.